AptosSDK/Src/Client/Services/BaseService.cs

```diff
@@ -0,0 +1,13 @@
+using Mirage.Aptos.SDK.DTO;
+
+namespace Mirage.Aptos.SDK.Services
+{
+	public class BaseService
+	{
+		protected string URL;
+		public BaseService(OpenAPIConfig config)
```

Comment:
add space

Reply:
Modern Rider formatting is confusing sometimes
<img width="412" alt="image" src="https://user-images.githubusercontent.com/38144180/209507201-789ac470-485c-4d36-822f-ee88c5a95a6e.png">

Reply:
Understandable